### ISSUE TITLE
pretraining using adam loss results and clearer loss pretraining results

### ISSUE THREAD

**Issue by Esaada** [NVIDIA/DeepLearningExamples#464]
Hi, can is there an option to share the pertaining loss for adam, as in this table:
https://github.com/NVIDIA/DeepLearningExamples/tree/master/TensorFlow/LanguageModeling/BERT#pre-training-accuracy-single-node
Second, Does the pertaining loss results on the 16gb gpus, are the same as those on the 32gb gpus? the results in the link above what experiment they belong to.

**Comment by swethmandava** [comment]
The results listed above are from pretraining using the LAMB optimizer. 
We don't have pretraining losses using Adam right now but provided scripts to run the full pretraining. We generally use SQuAD (or other finetuning tasks) accuracy to verify the goodness of our pretraining

**Comment by Esaada** [comment]
Thanks, are the final looses **suppose** to be the same with adam and lamb?
And does training with 16gb gpu will decrease accuracy because of the batchsize?

**Comment by swethmandava** [comment]
LAMB gave us better accuracies on SQuAD. 

Training with 16gb doesn't have to decrease accuracy - you can use gradient accumulation to mimic the same global batch size + recipe.
